testserver/testserver.go

```diff
@@ -282,6 +273,19 @@ func NewTestServer(opts ...testServerOpt) (TestServer, error) {
 		curTenantID:      firstTenantID,
 	}
 	ts.pgURL.set = make(chan struct{})
+
+	if err := ts.Start(); err != nil {
+		return nil, err
+	}
+
+	if ts.PGURL() == nil {
```

Comment:
I don't think this call is necessary (we set the `pgurl` above) but feel free to keep it in.